MapboxNavigation/RoutePageViewController.swift

```diff
@@ -56,13 +56,13 @@ class RoutePageViewController: UIPageViewController {
 extension RoutePageViewController: UIPageViewControllerDataSource, UIPageViewControllerDelegate {
     func pageViewController(_ pageViewController: UIPageViewController, viewControllerAfter viewController: UIViewController) -> UIViewController? {
         let controller = viewController as! RouteManeuverViewController
-        let stepAfter = maneuverDelegate.stepAfter(controller.step)
+        let stepAfter = maneuverDelegate.stepAfter(controller.step!)
```

Comment:
I realized when running the tests that it is possible for `step` to be `nil` when the tests create a minimal RouteManeuverViewController. The tests were crashing originally when I failed to say `step?` in my changes. But we do continue to assume that another view controller that initializes RouteManeuverViewController would set the step. Open to ideas here.